jsk_topic_tools/scripts/and_node.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import rospy
+import std_msgs.msg
+
+
+class ANDNode(object):
```

Comment:
Is it possible to merge `and_node.py` and `or_node.py` into one file?
Or is it possible to create common class for these nodes?

I think both of these nodes are for binary logic operation, so the most parts are the same.

Reply:
OK. Created `boolean_node.py`.